dts/bindings/video/ovti,ov7725.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020, FrankLi Limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: OV7725 CMOS video sensor
+
+compatible: "ovti,ov7725"
+
+properties:
+    reset-gpios:
+      type: phandle-array
+      required: false
```

Comment:
Is it really optional? the driver seems to unconditionnaly request it.

Reply:
Yes, If reset is connected to the reset circuit of the hardware, there is no need to use GPIO to control it.
If the gpio name cannot be found in the driver, gpio reset will not be performed
```
char *gpio_name = DT_INST_GPIO_LABEL(0, reset_gpios);

if (gpio_name) {
		drv_data->reset_gpio = device_get_binding(gpio_name);
		.....
}

...

if (drv_data->reset_gpio) {
		ret = gpio_pin_configure(drv_data->reset_gpio,
					 drv_data->reset_pin,
					 GPIO_OUTPUT_ACTIVE |
					 drv_data->reset_flags);
		if (ret) {
			return ret;
		}

		gpio_pin_set(drv_data->reset_gpio,
			     drv_data->reset_pin,
			     0);
		k_sleep(K_MSEC(1));
		gpio_pin_set(drv_data->reset_gpio,
			     drv_data->reset_pin,
			     1);
		k_sleep(K_MSEC(1));
	}
```